tinyalsa: Fix tinyalsa plugin build issue for Android.

sguiriec opened the pull request:
TinyAlsa part of ALSA plugin was not compiling on Upstream version for
Android integration. This patch is fixing the compilation issue. It has been
tested under Android IA tree.

Signed-off-by: Sebastien Guiriec <sebastien.guiriec@intel.com>

Commit:
tinyalsa: Fix tinyalsa plugin build issue for Android.

TinyAlsa part of ALSA plugin was not compiling on Upstream version for
Android integration. This patch is fixing the compilation issue. It has been
tested under Android IA tree.

Signed-off-by: Sebastien Guiriec <sebastien.guiriec@intel.com>

sguiriec commented:
@mgaio, @gdenneul, @cc6565 please review.

cc6565 commented on tinyalsa/TinyAlsaSubsystemBuilder.cpp:
```diff
@@ -39,7 +39,7 @@ extern "C"
  */
 void PARAMETER_FRAMEWORK_PLUGIN_ENTRYPOINT_V1(CSubsystemLibrary *subsystemLibrary,
-                                              core::log::Logger&)
+                                              core::log::Logger& logger)
```
coding rule: space before & and not after ;-)

Commit:
tinyalsa: Fix Coding style issue.

Signed-off-by: Sebastien Guiriec <sebastien.guiriec@intel.com>

Commit:
CI: Add clang-format configuration

This patch introduces generic clang-format coding style for any new
update of Parameter Framework ALSA plugin.

Signed-off-by: Sebastien Guiriec <sebastien.guiriec@intel.com>

Commit:
clang-format: Update all the reference code with coding styles.

This patch is updating the full plugin source code according to
Parameter Framework coding rules with clang-format tool.

Signed-off-by: Sebastien Guiriec <sebastien.guiriec@intel.com>

sguiriec commented:
@mgaio, @gdenneul, @cc6565 please review.
Pull request updated with clang-format CI like for Parameter Framework Core.
Code is reformat accordingly.

sguiriec commented:
Need to be aligned with former tinyalsa branch.
